docs/src/man/parallel_approximations.md

```diff
@@ -0,0 +1,145 @@
+# Parallel Approximations
+
+A subset of the approximation algorithms are implemented in parallel in the 
+`LazySets.Parallel` module. In order to use parallel versions of the algorithms,
+you can write:
+
+```@example
+using LazySets
+import LazySets.Parallel
+
+# call a method implemented in parallel, for example:
+S = Ball2(ones(100), 1.0)
+Parallel.box_approximation(S)
+```
+
+Note that after importing or using `LazySets.Parallel`, the version of the function
+used must be fully qualified, eg. `LazySets.Approximations.box_approximation` for the
+sequential function or `LazySets.Parallel.box_approximation` for the parallel version.
+
+The parallelization strategy that is available uses processes. To set the number
+of processes use the flag `-p N` at julia startup, where `N` is the number of
+desired processes. For example, do
+
+```julia
+$ julia -p 4
+```
+to launch `4` additional local worker julia processes. Use the keyword `auto`,
+as in
+```julia
+$ julia -p auto
+```
+to launch as many workers as the number of local CPU threads.
+
+
+```@contents
+Pages = ["parallel_approximations.md"]
+Depth = 3
+```
+
+```@meta
+DocTestSetup = quote
+    using LazySets, LazySets.Parallel
+end
+```
+
+## Parallel interval hulls 
+
+Consider the symmetric interval hull approximation
```

Comment:
Suggestion:
```suggestion
Consider the [symmetric interval hull approximation](@ref Symmetric interval hull)
```
and remove `, a modified version of that given in the Introduction, and` below?